Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -113,75 +134,85 @@ class TransactionHistoryViewController: UIViewController {
         }
         self.transactionFilterDropDown.setSelectedItemIndex(0)
     }
-    
+
     @objc func reloadTxsForCurrentFilter() {
         var currentFilterItem = DcrlibwalletTxFilterAll
         if self.transactionFilterDropDown.selectedItemIndex >= 0 && self.transactionFilters.count > self.transactionFilterDropDown.selectedItemIndex {
             currentFilterItem = self.transactionFilters[self.transactionFilterDropDown.selectedItemIndex]
         }
         self.applyTxFilter(currentFilter: currentFilterItem)
```

Comment:
I think we can combine both `reloadTxsForCurrentFilter` and `applyTxFilter` functions into 1, i.e. `reloadTxsForCurrentFilter`.